pkg/connector/source.go

```diff
@@ -385,12 +389,19 @@ func (s *Source) sanitizeRecord(r *opencdc.Record) {
 	if r.Payload.After == nil {
 		r.Payload.After = opencdc.RawData{}
 	}
-
 	if r.Metadata == nil {
-		r.Metadata = opencdc.Metadata{}
+		r.Metadata = opencdc.Metadata{
```

Comment:
This is a performance improvement, the methods on the `Metadata` type are pretty slow and result in allocations.